src/components/play/PlayUI.js

```diff
@@ -1521,8 +1521,6 @@ class PlayUI extends Component {
     }
 
     onKeypress(e) {
```

Comment:
Why'd you do this? sure there's a reason. Could you delete `onKeypress` too?

Reply:
I guess I should, I just thought it might have some use in the future.